llms/mlx_lm/requirements.txt

```diff
@@ -1,5 +1,5 @@
 mlx>=0.4
 numpy
-transformers>=4.38.0
+git+https://github.com/huggingface/transformers.git
```

Comment:
This is a pre-release? If starcoder requires this, let's wait until it hits pypi and put the version instead.

Reply:
Correct, sure